core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -322,9 +322,9 @@
                  target="_blank"
                  rel="noopener"
                  [href]="report.link"
-                 [class]="'oppia-impact-report-button oppia-sidebar-menu-link e2e-mobile-test-sidebar-impact-report-button-${report.year}'">
+                 [class]="'oppia-impact-report-button oppia-sidebar-menu-link e2e-mobile-test-sidebar-impact-report-button-'+report.year">
```

Comment:
The e2e class should not include the report year. Please find another way to select the correct element based on the text it contains, as previously explained.